graphql/task_resolver.go

```diff
@@ -331,11 +331,11 @@ func (r *taskResolver) DisplayTask(ctx context.Context, obj *restModel.APITask)
 func (r *taskResolver) EstimatedStart(ctx context.Context, obj *restModel.APITask) (*restModel.APIDuration, error) {
 	t, err := obj.ToService()
 	if err != nil {
-		return nil, InternalServerError.Send(ctx, fmt.Sprintf("Error while converting task %s to service", *obj.Id))
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("while converting task %s to service", *obj.Id))
```

Comment:
`utility.FromStringPtr`
```suggestion
		return nil, InternalServerError.Send(ctx, fmt.Sprintf("while converting task '%s' to service", *obj.Id))
```